openapi/index.yaml

```diff
@@ -25,6 +25,11 @@ components:
           example: []
           items:
             $ref: "#/components/schemas/Error"
+    FiscalCodeType:
+      description: The citizen's fiscal code.
+      type: string
+      pattern: "^[A-Z]{6}[0-9LMNPQRSTUV]{2}[ABCDEHLMPRST][0-9LMNPQRSTUV]{2}[A-Z][0-9LMNPQRSTUV]{3}[A-Z]$"
+      example: "SPNDNL80R13C555X"
```

Comment:
Please remove the example that refer to Danilo :)

Reply:
Changed!

Reply:
For the record, that tax code is used a lot: https://github.com/search?q=org%3Apagopa+SPNDNL80R13C555X&type=code

Indeed we copied that from https://github.com/pagopa/io-functions-commons/blob/8cf65e0ea3e043313ba6c62f58a61d3aedd340df/openapi/definitions.yaml#L360